Resources/Public/Javascript/PageView/ImageManipulationControl.js

```diff
@@ -71,7 +71,7 @@ dlfViewerImageManipulationControl = function(options) {
      * @type {Element}
      * @private
      */
-    this.toolContainerEl_ = dlfUtils.exists(options.toolContainer) ? options.toolContainer: $('.tx-dlf-Toolbox')[0];
+    this.toolContainerEl_ = dlfUtils.exists(options.toolContainer) ? options.toolContainer: $('.tx-dlf-toolbox')[0];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found fs.exists with non literal argument at index 0](https://app.codacy.com/gh/kitodo/kitodo-presentation/pullRequest?prid=5109013)

Reply:
False positive. We don't use `fs` here.